vendoring/README.md

```diff
@@ -16,3 +16,11 @@ vendoring update
 ### [`matplotlib`](https://github.com/matplotlib/matplotlib/)
 
 We take just a subset of `font_manager.py` used for selecting fonts. See [LICENSE](../src/dymoprint/_vendor/matplotlib/LICENSE) for the license.
+
+## Coding style
+
+Test code using the following command:
+
+```bash
+pylint src --disable invalid-name,missing-module-docstring,missing-function-docstring,too-many-arguments,too-many-locals,import-error,line-too-long,too-many-instance-attributes,missing-class-docstring,too-many-statements,too-few-public-methods
+```
```

Comment:
This ended up in the wrong `README`.

Also, it'd be nicer to use use `ruff` + `pre-commit` instead of `pylint` with a long list of arguments like this.

Reply:
I'll remove it indeed, and will integrate `ruff` later on

Reply:
If you give me a few minutes I have done ruff and pre-commit and I will give you a branch that will make pre-commit pass